DBMS/DBMS2/assignment2.sql

```diff
@@ -0,0 +1,5 @@
+-- to get information about issued books in ascending order firs on issue date then members name
+
+SELECT b.issue_date ,b.accession_no,b.member_id,m.member_name,
```

Comment:
ACTION: Again we have completely ignored the cook_return table which has return_date column.